app/models/info_request.rb

```diff
@@ -641,21 +571,20 @@ def calculate_status(cached_value_ok=false)
     if cached_value_ok && @cached_calculated_status
       return @cached_calculated_status
     end
-    @cached_calculated_status = @@custom_states_loaded ? self.theme_calculate_status : self.base_calculate_status
+    @cached_calculated_status = @@custom_states_loaded ? theme_calculate_status : base_calculate_status
   end
 
   def base_calculate_status
-    return 'waiting_classification' if self.awaiting_description
-    return described_state unless self.described_state == "waiting_response"
+    return 'waiting_classification' if awaiting_description
+    return described_state unless described_state == "waiting_response"
     # Compare by date, so only overdue on next day, not if 1 second late
     return 'waiting_response_very_overdue' if
-    Time.now.strftime("%Y-%m-%d") > self.date_very_overdue_after.strftime("%Y-%m-%d")
+    Time.now.strftime("%Y-%m-%d") > date_very_overdue_after.strftime("%Y-%m-%d")
     return 'waiting_response_overdue' if
-    Time.now.strftime("%Y-%m-%d") > self.date_response_required_by.strftime("%Y-%m-%d")
+    Time.now.strftime("%Y-%m-%d") > date_response_required_by.strftime("%Y-%m-%d")
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.